interface/src/app/analyze/experiment/experiment.js

```diff
@@ -4,87 +4,89 @@ angular.module('adage.analyze.experiment', [
   'ngResource'
 ])
 
-.factory('Experiment', ['$resource', function($resource) {
-  var Experiment = $resource(
-    '/api/v0/experiment/:accession/',
-    // TODO need to add logic for handling pagination of results.
-    // then, can change "limit" below to something sensible
-    {limit: 0}
-  );
-  Experiment.makeHref = {
-    // These functions create formatted URLs for direct linking to source
-    // material on ArrayExpress. They are attached to the Experiment factory
-    // so they can be made available in the same places that Experiment data
-    // are shown.
-    mlDataSource: function(experimentId, mlDataSource) {
-      return ('http://www.ebi.ac.uk/arrayexpress/files/{expId}/' +
-        '{expId}.raw.1.zip/{mlDataSource}')
-        .replace(/{expId}/g, experimentId)
-        .replace(/{mlDataSource}/g, mlDataSource);
-    },
-    arrExpExperiment: function(experimentId) {
-      return ('http://www.ebi.ac.uk/arrayexpress/experiments/{id}/')
-        .replace(/{id}/g, experimentId);
-    }
-  };
-  return Experiment;
-}])
+// // moved to app/experiment.service.js
```

Comment:
It is just a copy/paste without any code change, right?

Reply:
Right. The only thing different there is that the `angular.module` statement at top can be a little simpler since this service only needs `ngResource` and `adage.utils`.

Obviously this whole `experiment.js` file will be removed when I'm done. I found it helpful to remind myself where things were going as I refactored.